osu.Game.Rulesets.Osu.Tests/TestSceneOsuModPerfect.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Rulesets.Osu.Mods;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Tests.Visual;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Tests
```

Comment:
For consistency with `TestSceneOsuModDifficultyAdjust`:
```suggestion
namespace osu.Game.Rulesets.Osu.Tests.Mods
```
Goes for all of the others if we're making this a convention.